Entities/Industry/Drill/Drill.as

```diff
@@ -445,10 +445,14 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 
 	if (customData == Hitters::water)
 	{
-		s16 current_heat = this.get_u8(heat_prop) - heat_max * heat_reduction_water;
-		if (current_heat < 0) current_heat = 0;
+		s16 current_heat = this.get_u8(heat_prop);
```

Comment:
this should be a u8

Reply:
I applied the change and tested for a bit in offline. 
There should be no problems from changing f32 to u8 in line 136 and changing s16 to u8 in line 448.